<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器-黄然-2022414040115</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="2">
       <widget class="QPushButton" name="btnC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnCheng">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>×</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnJian">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>➖</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnCE">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnDS">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btn2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnDian">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btnZF">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>±</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btn0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnBFH">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnPF">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>x²</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnChu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>÷</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnDY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btn3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnJia">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnDel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>⌫</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnGH">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>√</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLineEdit" name="display">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
    width: 100px;
    height: 50px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btn6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: rgba(50,50,50,100);
   font-size: 36pt;
   color:while;
   border-radius: 2px;
}
QPushButton:hover{
 background-color: lightgreen;
color: black;
}
QPushButton:pressed{
  background-color:red;
 color: black;
}
QLineEdito{
font-size:40pt;
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
